tests/test_utils.py

```diff
@@ -298,6 +298,47 @@ def test_minify_pddl_problem():
         example03) == """((in-closet sweatpants1))"""
 
 
+def test_flatten_pddl_problem():
```

Comment:
nice looking tests! if you do the combination I suggested, please still keep the tests (just move them)